warp10/src/main/java/io/warp10/standalone/Warp.java

```diff
@@ -560,6 +561,13 @@ public static void main(String[] args) throws Exception {
     }
 
     WarpDist.setInitialized(true);
+    System.out.println("## Your Warp 10 setup:");
```

Comment:
Why stdout when most of stdout outputs have been replaced by logging calls?

Why those specific parameters? What is the purpose of these outputs?

Reply:
The purpose of this PR is to display some important metrics for platform sizing like the JVM heap or max fetch and ops.